### PR TITLE
Review error listing

### PR DESCRIPTION
**Pull request by bmallred**
Resolves: #2652 

image

### CODE REVIEW

**Comment by floresj** [comment]
This is looking pretty awesome. Just noticed two things. 

Identifying Information
- Contact Information not showing up
Your History
- Where you went to school not showing up

**Comment by bmallred** [comment]
Cool I'll check those out

**Comment by bmallred** [comment]
@floresj does education show up if you say yes to one of the branch questions? otherwise it is never suppose to show up on the review screen

**Comment by floresj** [comment]
:taco: 
selection_792

**Comment by floresj** [comment]
:shallow_pan_of_food: 
selection_793

File: src/components/Section/SubsectionElement.jsx [hunk 1]
```diff
@@ -26,6 +26,13 @@ export default class SubsectionElement extends React.Component {
   guid () {
     return newGuid()
   }
+
+  dataAttributes (props = {}) {
```
Comment (floresj) [inline src/components/Section/SubsectionElement.jsx]: That's helpful. Differentiating between data attrs and props.

**Pull request merged**